topics/command-tips.md

```diff
@@ -0,0 +1,88 @@
+# Command tips
+
+An @array-reply of strings, command tips are meant to provide additional information about commands for clients and proxies.
+This infromation should help clients and proxies to determine how to handle this command when Redis is in clustering mode.
+Unlike the command flags (the third element in `COMMAND`'s reply), which are internal in nature (Redis itself uses them internally), command tips are just exposed via `COMMAND` and Redis does not use them.
+
+The tips are arbirary strings, but in order to create some sort of concensus, here is a list of suggested tips:
+
+## Boolean tips
```

Comment:
I find the grouping of tips by their format/associated value a bit wierd. I think for a user, a single flat table with tips, their format/value and a description will be more usable.

Reply:
one reason why they're grouped is because the request and reply policies have default that defines the behavior of these aspects when no explicit flag is set.

Reply:
I see Guy listed the tips at the top, but not all combinations.
i.e. it lists `request_policy:<policy>` not `request_policy:all_shards`, `request_policy:few_shards`.
i'm ok with that, but i don't think that's what Yossi meant (if i understand what Yossi meant, i think it's problematic)